README.md

```diff
@@ -7,21 +7,98 @@ log-warper
 [![Stackage Nightly](http://stackage.org/package/log-warper/badge/nightly)](http://stackage.org/nightly/package/log-warper)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-Auxilary logging library, which is wrapper over
-[hslogger](http://hackage.haskell.org/package/hslogger) but allows
-to keep logger name into monadic context, making logging less boilerplate.
+`log-warper` is a high level and featureful logging library with monadic interface.
 
-Key features:
+You can jump right into [introduction tutorial](https://github.com/serokell/log-warper/blob/master/examples/HowTo.md)
+to see how to write logging with `log-warper`.
 
-1. Output is colored :star:
-2. Supports logging initialization from `.yaml` configuration file
-3. Has wrapper for pure logging via `StateT`
-4. Supports log rotation
-5. Flexibles and easy setting up of loggers (using monoidal builders and lenses)
-6. Ability to acquire last `N` megabytes of logs from in-memory cache
+## Features
 
-Contributing
-------------
+Here is the list of feature `log-warper` provides.
+
+1. Hierarchical logger names.
+
+   Logger names (tags for loggers) forms hierarchy. It means, that `""`
+   (also known as `mempty` or `rootLoggerName`) is a parent of logger with name `"node"` which isDir
+   a parent of logger with name `"node.communication"`. So, logger name comprises dot-separated components.
+   This means that if some logger name doesn't have some settings (like severity or output file) it takes
+   its settings from its closest parent, containing this settings.
```

Comment:
twice "its" doesn't sound cool, probably "takes its settings form *the* closest parent" or vice versa